packages/web-app-files/src/components/AppBar/AppBar.vue

```diff
@@ -168,13 +164,9 @@ export default {
   mixins: [Mixins, MixinFileActions],
   setup() {
     return {
-      isPersonalLocation: watchActiveLocation(isLocationSpacesActive, 'files-spaces-personal-home'),
-      isPublicLocation: watchActiveLocation(isLocationPublicActive, 'files-public-files'),
-      isSpacesProjectsLocation: watchActiveLocation(
-        isLocationSpacesActive,
-        'files-spaces-projects'
-      ),
-      isSpacesProjectLocation: watchActiveLocation(isLocationSpacesActive, 'files-spaces-project')
+      isPersonalLocation: useActiveLocation(isLocationSpacesActive, 'files-spaces-personal-home'),
+      isPublicLocation: useActiveLocation(isLocationPublicActive, 'files-public-files'),
+      isSpacesProjectsLocation: useActiveLocation(isLocationSpacesActive, 'files-spaces-projects')
```

Comment:
```suggestion
      isSpacesProjectsLocation: useActiveLocation(isLocationSpacesActive, 'files-spaces-projects'),
      isSpacesProjectLocation: useActiveLocation(isLocationSpacesActive, 'files-spaces-project')
```

Needs this in order to be able to navigate into an individual space I think?

Reply:
Okay no actually doesn't, messed up pulling the latest state of this PR 🤦🏽